scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -1297,18 +1314,22 @@ async def action(page: object) -> None:
+            "prompt": prompt,
+            "marker": marker,
+            "required_text": required_text,
+            **(extra_details or {}),
+            **observed,
+        }
+        semantic_judge = getattr(exc, "semantic_judge", None)
+        if isinstance(semantic_judge, dict):
+            failure_details["semantic_judge"] = semantic_judge
         return _result(
             case_name,
             False,
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)